src/Salt/UserBundle/Controller/SignupController.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Salt\UserBundle\Controller;
+
+use Symfony\Bundle\FrameworkBundle\Controller\Controller;
+use Salt\UserBundle\Form\Type\SignupType;
+use Symfony\Component\HttpFoundation\Request;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Template;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Route;
+use Sensio\Bundle\FrameworkExtraBundle\Configuration\Method;
+use Salt\UserBundle\Entity\User;
+use Salt\UserBundle\Entity\Organization;
+use App\Command\User\AddUserCommand;
+use App\Command\User\AddOrganizationCommand;
+use App\Command\CommandDispatcherTrait;
+use Qandidate\Bundle\ToggleBundle\Annotations\Toggle;
+
+/**
+ * Signup Controller.
+ *
+ * @Route("public/user")
+ * @Toggle("create_account")
+ */
+class SignupController extends Controller
+{
+    use CommandDispatcherTrait;
+
+    /**
+     * Creates a new user entity
+     *
+     * @Route("/signup", name="public_user_signup")
+     * @Method({"GET", "POST"})
+     * @Template()
+     *
+     * @param Request $request
+     *
+     * @return array|\Symfony\Component\HttpFoundation\RedirectResponse
+     */
+    public function signupAction(Request $request)
```

Comment:
The `signupAction` should check for the feature being enabled (and throw a `NotFoundHttpException` if it is not enabled)